src/Logic/SimpleMetaTagger.ts

```diff
@@ -583,6 +584,47 @@ export default class SimpleMetaTaggers {
         }
     )
 
+    private static currency = new InlineMetaTagger(
+        {
+            keys: ["_currency"],
+            doc: "Adds the currency valid for the object, based on country or explicit tagging. Can be a single currency or a semicolon-separated list of currencies. Empty if no currency is found.",
+            isLazy: true,
+        },
+        (feature) => {
+            Utils.AddLazyProperty(feature.properties, "_currency", () => {
+                // Initialize a list of currencies
+                const currencies = {}
+
+                // Check if there are any currency:XXX tags, add them to the map
+                for (const key in feature.properties) {
+                    if (key.startsWith("currency:")) {
+                        if (feature.properties[key] === "yes") {
+                            currencies[key.slice(9)] = true
+                        } else {
+                            currencies[key.slice(9)] = false
+                        }
+                    }
+                }
+
+                // Determine the default currency for the country
+                const defaultCurrency = countryToCurrency[feature.properties._country.toUpperCase()]
```

Comment:
There is a small, technical problem here.

`_country` is loaded automatically, but requires a few HTTP-roundtrips to be determined.

In the case that the currency is needed straight away (e.g. to render an iconBadge),   `_country` will not be loaded yet.

The correct approach here is to:
- use the event store with properties - this is passed as parameter in the function
- install a callback on the properties, specifically to check if `country` is set
- if country is set, to determine the country
- if all is done, to detach the callback (by returning true)

I'll merge this PR and fix it though!